locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -49,6 +50,68 @@
 import javax.annotation.Nullable;
 import javax.xml.ws.WebServiceException;
 
+import org.apache.commons.lang3.ArrayUtils;
```

Comment:
@grkvlt just came across another import rearrange you did in 33b0b1c879a9ed07bcdc3138893cd8017e211cfa

can you configure your import order as per
http://brooklyn.apache.org/v/latest/dev/env/ide/index.html#intellij-idea ?